packages/cross_file/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
+
+* Stable null safety release.
+
 ## 0.3.0-nullsafety
```

Comment:
We're removing prerelease release notes, so rather than adding a section, just remove `-nullsafety` from what's there.

Reply:
That makes sense, thanks I updated the file.